inc/classes/optimization/class-cache-dynamic-resource.php

```diff
@@ -179,10 +179,12 @@ public function is_excluded_file( $src ) {
 			return true;
 		}
 
-		$file['query'] = remove_query_arg( 'ver', $file['query'] );
+		if ( is_array( $file ) && ! empty( $file['query'] ) ) {
+			$file['query'] = remove_query_arg( 'ver', $file['query'] );
 
-		if ( $file['query'] ) {
-			return true;
+			if ( $file['query'] ) {
```

Comment:
As this is the last conditional check in the method, we can eliminate the hard coded boolean return statements. How? Return the boolean state from the conditional expression. 

[See this quick tip](https://knowthecode.io/smelly-code-ifreturn-truefalse-code-pattern) for more info.